features/src/main/feature/camel-features.xml

```diff
@@ -1267,6 +1267,15 @@
 <!--        <bundle dependency='true'>wrap:mvn:com.github.jasminb/jsonapi-converter/${jasminb-jsonapi-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-jsonapi/${project.version}</bundle>-->
 <!--    </feature>-->
+        <feature name='camel-json-patch' version='${project.version}' start-level='50'>
+            <feature version='${camel.osgi.version.range}'>camel-core</feature>
+            <feature version='[2.16,2.17)'>jackson</feature>
+            <bundle dependency='true'>wrap:mvn:com.github.java-json-tools/json-patch/${json-patch-version}$overwrite=merge&amp;Export-Package=com.github.fge*;version=${json-patch-version}&amp;Import-Package=com.fasterxml*,com.github*;javax*</bundle>
+            <bundle dependency='true'>wrap:mvn:com.github.java-json-tools/jackson-coreutils/2.0$overwrite=merge&amp;Export-Package=com.github.fge*;version=2.0&amp;Import-Package=com.fasterxml*,com.github*;javax*</bundle>
```

Comment:
Peeked the fix for this bundles from camel-openstack
https://github.com/jbonofre/camel-karaf/pull/143/files#diff-8cca327d8104823fc5bc3583bb826255c61052b3b902b59d0b1ba5bc0fe9342eR1518